components/blocks/content.tsx

```diff
@@ -23,6 +24,11 @@ const sizeClasses = {
   "2xl": "prose-2xl",
 };
 
+export const paddingOptions = {
+  "Mobile: No Padding": "pt-0 px-0 md:px-8",
+  "Mobile: No Top and Horizontal Padding": "pt-0 px-0 md:px-8",
+};
```

Comment:
These are the same I think. Can we expose the padding options from the container i.e. a field using values from `verticalPadding` from `container.tsx`?

Reply:
Might be annoying with the `md:` though, so not sure

Reply:
We have some break points here specific for mobile, and I don't want to break things on other pages, what do you think?

Reply:
done